experiments/SlabLand/README.md

```diff
@@ -0,0 +1,161 @@
+
+# Atmos Single Column & Slab Land
+
+
+
+
+
+
+            |============================== z = 1e4 m  
+        A   |
+        T   |              ...
+        M   |
+        O   |------------------------------ z = 1e3 m    
+        S   |
+            |
+            |============================== z = 0 m    
+
+
+       LAND |============================== z = 0 m  
+
+
+## **Atmos Component**
+
+### 1. Formulation
+We're using the ClimateMachine.jl's Atmos SingleStack configuration. In this simplified case, we are solving the 5 diffusion equations: 1 for density, 3 for the velocity components, and 1 for a tracer. 
+
+$$
+\frac{\partial ρ}{\partial t} = - ∇ \cdot (\vec{u} ρ + μ ∇ ρ) \qquad 
+\frac{\partial \vec{u}}{\partial t} = - ∇ \cdot (\vec{u} \vec{u} + ν ∇ \vec{u}) \qquad 
+\frac{\partial θ}{\partial t} = - ∇ \cdot (\vec{u} \theta + κ ∇ θ) \qquad 
+$$
+with μ, ν and κ representing the respective viscosity and diffusivity coefficients. The boundary conditions are:
+- Upper (external) boundary:
+    - ρ: `Impenetrable()` 
+    - u: `FreeSlip()`
+    - θ: `Insulating()`
+- Lower (coupled) boundary:
+    - ρ: `Impenetrable()`  
+    - u: `FreeSlip()`
+    - θ: `CoupledPrimary()`: $\frac{\partial θ}{\partial t} = - ∇ \cdot F_{tot}$ where $F_{tot}$ is the surface flux calculated by `calculate_land_sfc_fluxes(_...)` using Atmos and Land properties (see below) 
```

Comment:
Is the boundary condition $$\partial θ / \partial n = F_{tot}$$? (not the divergence of the flux)

Reply:
Not quite, the way we set the BCs in our case is ultimately via the normal flux, i.e. `fluxᵀn.ρθ += - F_tot / scaling_params` (referred to as `local_flux` in different parts of the code), and then the model calculates the div of that.

Reply:
ok! that is what I thought. I was just commenting on the way it is written. (the math expression does not reflect the BC, which is a Neumann BC on flux like you wrote  - the math instead reflects the entire RHS at the boundary).